main.tf

```
@@ -51,105 +55,49 @@ resource "aws_iam_role" "this" {
   tags = var.tags
 }
 
-resource "aws_iam_role_policy_attachment" "assume_roles_attach" {
-  policy_arn = aws_iam_policy.assume_roles.arn
-  role       = aws_iam_role.this.name
-}
-
-# Allow the runtime to assume roles in the /sym/ path in safelisted accounts
 resource "aws_iam_policy" "assume_roles" {
   name = local.role_name
   path = "/sym/"
 
-  description = "Base permissions for the Sym runtime"
+  description = "These are base permissions required for the Sym Runtime to perform any actions in your AWS account. This policy allows the Sym Runtime to assume roles in the /sym/ path in safelisted accounts."
```

```suggestion
  description = "These are base permissions required for the Sym Runtime to perform any actions in your AWS account. This policy allows the Sym Runtime to assume roles in the /sym/ path in the current AWS account."
```

or something? since there isn't actually a safelist anymore?